renovate.json

```diff
@@ -2,9 +2,11 @@
   "extends": ["config:base"],
   "ignorePaths": ["packages/build-field-tyes/__fixtures__/**/*"],
   "lockFileMaintenance": { "enabled": true },
-  "separateMinorPatch": true,
+  "separateMinorPatch": false,
+  "separateMajorMinor": true,
   "prConcurrentLimit": 0,
-  "baseBranches": ["renovating"],
+  "prHourlyLimit": 3,
```

Comment:
Additionally added this restriction to prevent renovate from overwhelming our GitHub action limits